whylog/teacher/user_intent.py

```diff
@@ -1,10 +1,22 @@
+class Group(object):
+    def __init__(self, group_content, group_converter):
+        self.content = group_content
+        self.converter = group_converter
+
```

Comment:
I don't understand question, but to clarify, here is an example:
line_content could be "12/03/2009 Error on comp1"
group_content could be then "12/03/2009" or "comp1"